apollo-compiler/src/main/kotlin/com/apollostack/compiler/ir/CodeGenerator.kt

```diff
@@ -3,5 +3,5 @@ package com.apollostack.compiler.ir
 import com.squareup.javapoet.TypeSpec
 
 interface CodeGenerator {
-  fun toTypeSpec(): TypeSpec
-}
\ No newline at end of file
+  fun toTypeSpec(irPkgName: String): TypeSpec
```

Comment:
I see `pkg` here and `package` in other places. Do mind using `package` everywhere for consistency? Eg.: `irPackageName`